modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -246,6 +246,13 @@ output application/json encoding="sjis"
 "～―－＄￠￡㈱①"
 ----
 
+====  The multipart/form-data reader doesn't support UTF-8 characters by default
```

Comment:
This header should match the format of the other sections, also note that the sections are itemized above.